### PR TITLE
add a node.js yaml config file

### PR DESCRIPTION
**Pull request by lukaswski**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
```
Comment (oneacik) [inline .github/workflows/node.js.yml]: Please don't use windows images.
Probably on linux cypress would run too.

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        node-version: [14.x, 15.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
```
Comment (oneacik) [inline .github/workflows/node.js.yml:28]: nice

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    - run: npm run build --if-present
+    - run: npm test -- -u
```
Comment (oneacik) [inline .github/workflows/node.js.yml:30]: what u does?

File: .github/workflows/node.js.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (oneacik) [inline .github/workflows/node.js.yml:10]: does it mean it will run when PR is pointed to master?
Or will it run if PR is from master to '*' ?

**Comment by lukaswski** [comment]
Sorry for this mess, I'm just checking, what happend if i change something in yaml file, becausecose i'm looking for resolve of cypress problem

**Pull request closed without merging**